run_cross_test.py

```diff
@@ -15,43 +15,25 @@
 # limitations under the License.
 
 '''Runs the basic test suite through a cross compiler.
-Not part of the main test suite because of two reasons:
 
-1) setup of the cross build is platform specific
-2) it can be slow (e.g. when invoking test apps via wine)
+This is now just a wrapper around run_project_tests.py with specific arguments
+'''
```

Comment:
Nice! But at least soon after this PR I'd like to see this wrapper go and the CI scripts just do the right thing instead.